app/views/pages/articles/2016-11-17-ruby-au-elections.html.md

```diff
@@ -0,0 +1,48 @@
+# Ruby Au Elections
+
+
+We have 5 positions open, President and Secretary, as well as two full-term and one half-term general position. Below is a brief description of each position, and a link to a nomination form to help streamline the meeting. If you are nominating someone other than yourself, please be sure to check with them first.
```

Comment:
5 open positions which include the President and Secretary roles, and two full-term and one half-term general positon.